docs/build/program/run.md

```diff
@@ -16,7 +16,7 @@ After saving your [code sample](/build/program/#hello-world-the-code-sample-tab)
 
 {{< readfile "/static/include/program/authenticate.md" >}}
 
-## Run Code Remotely
+## Run code remotely
```

Comment:
I'm noticing that this section could also use improvement. I'm happy to make a ticket unless you want to roll it into here.

Reply:
ticket pls